syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTOutBailmentReply";
option optimize_for = LITE_RUNTIME;

message OutBailmentReply {
    optional uint32 version = 1;
    optional string instructions = 2;
}
